src/components/Login/Login.js

```diff
@@ -82,6 +89,13 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
 
   const formMessage = useMemo(() => getForgotErrorMessage(location), [location]);
 
+  const activationInProgress = useMemo(() => isActivactionInProgress(location), [location]);
+
+  if(activationInProgress && typeof(window.gtag) === 'function') {
+    const gtag = window.gtag;
+    gtag('event', 'conversion', {'send_to': 'AW-1065197040/ZA62CKv_gZEBEPC79vsD'});
```

Comment:
Dis you tested it manually? Does it generate a remote request?

Reply:
yes, it's generated a remote request

Reply:
Great!